apps/range-voting/app/components/VoteStatus.js

```diff
@@ -19,15 +20,20 @@ const ATTRIBUTES = {
     Icon: IconCheck,
     color: theme.positive,
   },
+  [VOTE_STATUS_EXECUTED]: {
+    label: 'Approved and executed',
+    Icon: IconCheck,
+    color: theme.positive,
+  },
   [VOTE_STATUS_FAILED]: {
     label: 'Rejected',
     Icon: IconCross,
     color: theme.negative,
   },
 }
 
-const VoteStatus = ({ vote: { data, support, quorum } }) => {
-  const status = getVoteStatus(data, support, quorum)
+const VoteStatus = ({ vote: vote }) => {
```

Comment:
Shouldn't it work with `const VoteStatus = ({ vote }) => {` ?

Reply:
I think that was giving me issues but feel free to try it.